### ISSUE TITLE
uber-Juno keeps updating Julia at every start and it takes ages.

### ISSUE THREAD

**Issue by ramzeee** [JunoLab/Juno.jl#595]
Every i start Julia over Atom and uber-Juno it does an update, the download goes fast and then i need to wait 10 minutes until the fetching is ready. What can i do to prevent or avoid this?

**Comment by pfitzseb** [comment]
What exactly is it downloading?

**Comment by ramzeee** [comment]
Its atom or juno is updating Julia:
![image](https://user-images.githubusercontent.com/67923653/86909841-04608900-c119-11ea-9d88-05fd1df11830.png)

**Comment by timholy** [comment]
I can't answer why it's updating packages, but for the record, for me package updates take a few seconds, not 10 minutes. Let me guess: you're on Wndows and you have antivirus software running. Search [discourse](https://discourse.julialang.org/) for some suggestions.

**Comment by pfitzseb** [comment]
Yes, that's the most likely cause for that specific issue. The fact that we're re-instantiating the required dependencies on every start is very weird though. Can you confirm that `C:\users\rspro\.julia` is persistent?

**Comment by ramzeee** [comment]
Yes I am on windows, its an office computer, so i cant change the antivirus so easily. I think that it is so slow has to do with[ this](https://discourse.julialang.org/t/julia-hangs-after-attempt-at-pkg-add-in-linux-cluster/4320/2) problem, which i also have. But i can live with that because i can download packages before going on a break or overnight.  But Starting Atom and Juno happens more often. Besides the long time it takes, i also would just like to control updates manually if possible. 

The folder C:\users\rspro\.julia exists and has subfolders "logs" and "registries" in which all sub-folders and files have a time stamp from the date of first installation of julia 11 May. So whatever it is downloading or updating, it seems not to be written in this folder. Which seems weird to me.

**Comment by pfitzseb** [comment]
Is (was) this a JuliaPro installation at some point?

**Comment by ramzeee** [comment]
No sorry the username is rspro multiple people working on the same account on this project. 
But it was a normal Julia installation, i never heared of JuliaPro until now.

**Comment by ramzeee** [comment]
> 
> 
> I can't answer why it's updating packages, but for the record, for me package updates take a few seconds, not 10 minutes. Let me guess: you're on Wndows and you have antivirus software running. Search [discourse](https://discourse.julialang.org/) for some suggestions.

I now excluded the user\.julia folder but it didn' speed up. 
I tried uninstalling and reinstalling. No change it is still painfully slow.
